Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1719,12 +1722,14 @@ void RenderElement::imageContentChanged(CachedImage& cachedImage)
         if (cachedImage.hasPaintedHDRContent())
             document().setHasPaintedHDRContent();
     }
+
+    if (document().hasPaintedHDRContent()) {
```

Comment:
Every time I read "hasPaintedHDRContent" I have to think "Does it mean 'painted HDR content sometime in the past', or 'owns HDR content which is painted'".

Reply:
Yes you are right. The names should be cleaned such that we do not have `hasPaintedHDRContent` and `drawsHDRContent` etc. I do that in a separate PR.

Reply:
The naming is addressed in https://github.com/WebKit/WebKit/pull/42556.